src/index.js

```diff
@@ -312,7 +315,14 @@ class SoshContentScript extends ContentScript {
     if (this.store.userCredentials != undefined) {
       await this.saveCredentials(this.store.userCredentials)
     }
-
+    if (await this.isElementInWorker('#password')) {
+      await this.evaluateInWorker(function reload() {
```

Comment:
why this reload ?

Reply:
Because on some execution, the login page load but stay on the form, reloading seems to set this up correctly